api/resources_portal/test/views/test_user.py

```diff
@@ -18,12 +18,24 @@ class TestUserListTestCase(APITestCase):
     """
 
     def setUp(self):
+        self.user = UserFactory()
+        self.superuser = User.objects.create_superuser("new_user", "newemail@test.com", "password")
         self.url = reverse("user-list")
         self.user_data = factory.build(dict, FACTORY_CLASS=UserFactory)
         # Don't post created_at and updated_at.
         self.user_data.pop("created_at")
         self.user_data.pop("updated_at")
 
+    def test_list_request_from_admin_succeeds(self):
+        self.client.force_authenticate(user=self.superuser)
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_list_request_from_non_admin_forbidden(self):
```

Comment:
Deepa said:

> We should let non-admin users to retrieve users. I can see a use case where people can request to be added to the organization too.

and I believe she was referring to list. Non-admin users need to be able to find other users for invitations and such.